CoinUtils error

Following the manual on compilation I am consistently ending up in this error. It will also fail for Clp and Osi.

What would be the way to force a Coin build from the AliceVision repo?

```
CMake Error at src/CMakeLists.txt:433 (find_package):
  By not providing "FindCoinUtils.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "CoinUtils", but CMake did not find one.

  Could not find a package configuration file provided by "CoinUtils" with
  any of the following names:

    CoinUtilsConfig.cmake
    coinutils-config.cmake

  Add the installation prefix of "CoinUtils" to CMAKE_PREFIX_PATH or set
  "CoinUtils_DIR" to a directory containing one of the above files.  If
  "CoinUtils" provides a separate development package or SDK, be sure it has
  been installed.
```

@simogasp that does not do the trick yet. I am able to install the osi_clp repo, but Clp by itself does not.

```
cmake -DCMAKE_INSTALL_PREFIX=/opt/alicevision ..
CMake Error at CMakeLists.txt:36 (find_package):
  By not providing "FindOsi.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Osi", but
  CMake did not find one.

  Could not find a package configuration file provided by "Osi" with any of
  the following names:

    OsiConfig.cmake
    osi-config.cmake

  Add the installation prefix of "Osi" to CMAKE_PREFIX_PATH or set "Osi_DIR"
  to a directory containing one of the above files.  If "Osi" provides a
  separate development package or SDK, be sure it has been installed.
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -46,7 +46,7 @@ Other optional libraries can enable specific features (check "CMake Options" for
 * UncertaintyTE (Uncertainty computation)
 * CoinUtils >= 2.9.3 (if you need to build it use [our fork](https://github.com/alicevision/CoinUtils) with a CMake build system)
 * Coin-or linear programming (Clp) (if you need to build it use [our fork](https://github.com/alicevision/Clp) with a CMake build system)
-* Open Solver Interface (Osi) >= 0.106.10 (if you need to build it use [our fork](https://github.com/alicevision/osi_clp) with a CMake build system)
+* Open Solver Interface (Osi) >= 0.106.10 (if you need to build it use [our fork](https://github.com/alicevision/Osi)
 
 
 AliceVision also depends on some embedded libraries:

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -44,20 +44,23 @@ Other optional libraries can enable specific features (check "CMake Options" for
 * PCL (Point Cloud Library) >= 1.12.1 for the registration module
 * PopSift (feature extraction on GPU)
 * UncertaintyTE (Uncertainty computation)
+* CoinUtils >= 2.9.3 (if you need to build it use [our fork](https://github.com/alicevision/CoinUtils) with a CMake build system)
+* Coin-or linear programming (Clp) (if you need to build it use [our fork](https://github.com/alicevision/Clp) with a CMake build system)
+* Open Solver Interface (Osi) >= 0.106.10 (if you need to build it use [our fork](https://github.com/alicevision/osi_clp) with a CMake build system)
+
 
 AliceVision also depends on some embedded libraries:
 
-* CoinUtils >= 2.9.3 (internal)
-* Coin-or linear programming (Clp) (internal)
 * Flann >= 1.8.4 (internal)
 * Lemon >= 1.3 (internal)
 * MeshSDFilter (internal)
 * OpenMesh (internal)
-* Open Solver Interface (Osi) >= 0.106.10 (internal)
+
+
 
 Building the project using vcpkg (recommended on Windows)
 --------------------------------
-[Vcpkg](https://github.com/alicevision/vcpkg) is a tool that helps in acquiring, building, and managing C/C++ libraries.
+[Vcpkg](https://github.com/alicevision/vcpkg) is a package manager that helps in acquiring, building, and managing C/C++ libraries.
 AliceVision's required dependencies can be built with it. Follow the [installation guide](https://github.com/alicevision/vcpkg/blob/alicevision_master/README.md#quick-start-windows) to setup vcpkg.
 
 ```bash
@@ -192,9 +195,6 @@ At the end of the cmake process, a report shows for each library which version (
 -- EIGEN: 3.3.4
 -- CERES: 1.10.0
 -- FLANN: 1.8.4 (external)
--- CLP: 1.15.11 (internal)
--- COINUTILS: 2.9.3 (internal)
--- OSI: 0.106.10 (internal)
 -- LEMON: 1.3 (internal)
 ```
 

